### DISCUSSION TITLE
risc-v support

### DISCUSSION THREAD

**Discussion by mvysny** [grafana/grafana#69332] [General]
Thank you for such a wonderful tool, I'm happily running Grafana on my Raspberry PI 3B+. However, the RISC-V architecture is gaining momentum and Ubuntu just announced official support for a couple of devices. For my experiments I've got myself the LicheeRV Dock device with 512MB worth of RAM. Installing Ubuntu 22.04 was quite easy: https://wiki.ubuntu.com/RISC-V/LicheeRV .

Unfortunately I've found out that Grafana doesn't support RISC-V out-of-the-box. Would it be possible for you to cross-compile Grafana for RISC-V? You do not have to own the RISC-V device to test: it's quite easy to run Ubuntu RISC-V in a QEMU emulator as described at https://wiki.ubuntu.com/RISC-V/QEMU .

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82633. Feel free to continue the discussion around this there. Thank you!
